boxwise_flask/type_defs.py

```diff
@@ -11,25 +11,59 @@
         allUsers: [User]
         user(email: String): User
     }
+
+    type Mutation {
+        createBox(box_creation_input:CreateBoxInput):Box
+    }
+
     type Base {
         id: Int
         name: String
-        currencyname: String
-        organisation_id: Int
+        currencyName: String
+        organisationId: Int
     }
-    type User{
+
+    type User {
         id: Int!
         organisation_id: Int
         name: String
         email: String!
-        cms_usergroups_id: Int
+        usergroup_id: Int
         valid_firstday: Date
         valid_lastday: Date
-        camp_id: [Int]
+        base_id: [Int]
         lastlogin: Datetime
         lastaction: Datetime
     }
 
+    type Box {
+        id: Int
+        box_id: String!
+        product_id: String
+        size_id: String
+        items: String
+        location_id: String
```

Comment:
location id is a string in the box but an int in creation data should they be the same?